src/main/java/com/eternalcode/combat/fight/tagout/TagOutCommand.java

```diff
@@ -0,0 +1,97 @@
+package com.eternalcode.combat.fight.tagout;
+
+import com.eternalcode.combat.config.implementation.PluginConfig;
+import com.eternalcode.combat.notification.NotificationAnnouncer;
+import com.eternalcode.combat.util.DurationUtil;
+import dev.rollczi.litecommands.argument.Arg;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.entity.Player;
+import panda.utilities.text.Formatter;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.UUID;
+
+@Route(name = "tag-out", aliases = "tag")
+public class TagOutCommand {
+
+    private final FightTagOutService fightTagOutService;
+    private final NotificationAnnouncer announcer;
+    private final PluginConfig config;
+
+    public TagOutCommand(FightTagOutService fightTagOutService, NotificationAnnouncer announcer, PluginConfig config) {
+        this.fightTagOutService = fightTagOutService;
+        this.announcer = announcer;
+        this.config = config;
+    }
+
+    @Execute(required = 1)
+    @Permission("eternalcombat.tagout")
+    void tagout(Player sender, @Arg Duration time) {
+        UUID targetUniqueId = sender.getUniqueId();
+
+        Formatter formatter = new Formatter()
+            .register("{PLAYER}", sender.getName())
+            .register("{TIME}", DurationUtil.format(time));
+
+        this.fightTagOutService.tagOut(targetUniqueId, time);
+
+        String format = formatter.format(this.config.messages.admin.adminTagOutSelf);
+        this.announcer.sendMessage(sender, format);
+    }
+
+    @Execute(required = 2)
+    @Permission("eternalcombat.tagout")
+    void tagout(Player sender, @Arg Player target, @Arg Duration time) {
+        UUID targetUniqueId = target.getUniqueId();
+
+        Instant now = Instant.now();
+        Duration remaining = Duration.between(now, now.plus(time));
+
+        Formatter formatter = new Formatter()
+            .register("{PLAYER}", target.getName())
+            .register("{TIME}", DurationUtil.format(remaining));
+
+        this.fightTagOutService.tagOut(targetUniqueId, time);
+
+        String adminTagOutFormat = formatter.format(this.config.messages.admin.adminTagOut);
+        this.announcer.sendMessage(sender, adminTagOutFormat);
+
+        String playerTagOutFormat = formatter.format(this.config.messages.admin.playerTagOut);
+        this.announcer.sendMessage(target, playerTagOutFormat);
+    }
+
+    @Execute(route = "remove", required = 1)
+    @Permission("eternalcombat.tagout")
+    void untagout(Player sender, @Arg Player target) {
+        UUID targetUniqueId = target.getUniqueId();
+
+        this.fightTagOutService.unTagOut(targetUniqueId);
+
+        Formatter formatter = new Formatter()
+            .register("{PLAYER}", target.getName());
+
+        if (targetUniqueId != sender.getUniqueId()) {
```

Comment:
```suggestion
        if (targetUniqueId.equals(sender.getUniqueId())) {
```

Reply:
![image](https://github.com/EternalCodeTeam/EternalCombat/assets/80779749/43cf015a-7b14-4e59-81f6-6782a0b027b5)

This works, if You target Yourself, you will be sent playerUntagOut ("You are now taggable!")
If you target somebody else, you will have admin message ("Successfully enabled tag for &e{PLAYER}!") and the target will get playerUntagOut

Reply:
Yes, I understood that. I meant how do you check the player's UUID. When comparing UUIDs with ==(!=) you check if they are in the same reference (memory location), so UUIDs should be checked using the equals method. Theoretically, checking with == may work, but using equals is more readable.
```suggestion
        if (!targetUniqueId.equals(sender.getUniqueId())) {
```